### PR TITLE
Home and Category Pages

### PR DESCRIPTION
**Pull request by yaswanthmyneni**
Added Home and Category pages.

home link: https://quirky-pasteur-fd2cf5.netlify.app/
category link: https://quirky-pasteur-fd2cf5.netlify.app/quiz/category/category.html

### CODE REVIEW

File: home.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        <div class="navbar-container">
+            <a class="navbar navbar-m-lr-1" href="#">
+                <h1 class="navbar-m-0">LUCENT QUIZ</h1>
+            </a>
+            <a class="navbar-m-left-auto navbar" href="#">
+                <h5>LOGIN</h5>
+            </a>
+            <a class="navbar navbar-m-lr-1" href="#">
+                <i class="fa-brands fa-github"></i>
+            </a>
```
Comment (Rishabh-078) [inline home.html]: The gap between Login & Github can be increased, just from a UI point of view.

File: home.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        <div>
+            <a href="#">
+                <i class="fa-brands fa-github social-links"></i>
+            </a>
+            <a href="#">
+                <i class="fa-brands fa-linkedin social-links"></i>
+            </a>
+            <a href="#">
+                <i class="fa-brands fa-twitter social-links"></i>
+            </a>
```
Comment (malaykhakhar) [inline home.html:91]: There is a small blue colored dash next to the twitter and linkedin logos in the footer.

**Review by malaykhakhar — COMMENTED** [review]
The code looks pretty clean. You can merge it after rectifying the footer icon issue.

**Review by Sahil25061999 — APPROVED** [review]
Everything looks good except the padding on the right of navbar is bit small

**Pull request merged**